.github/workflows/build.yml

```diff
@@ -65,19 +65,21 @@ jobs:
         activate-environment: test
         channels: conda-forge
         python-version: ${{ matrix.python-version }}
-        # this is needed for caching to work properly:
-        use-only-tar-bz2: true
 
     - name: Conda info
       run: conda info --all
 
     - name: Install build dependencies
       run: |
+        # NOTE: C-compiler was pinned because newer versions were causing
+        #       failures on github's Actions with the Ubuntu runner. Cause of
+        #       the failures is unknown, so might be worth trying unpinning
+        #       this in the future.
         conda install --quiet --yes \
-            c-compiler \
+            "c-compiler=1.5.1" \
             coverage \
             cython \
-            lalsuite \
+            "lalsuite" \
```

Comment:
I guess this is un-needed and a testing hangup?

Reply:
Yeah, I was testing with pinned versions believing lalsuite to be the issue, but instead the compiler was causing a problem